src/gui/transferlistwidget.h

```diff
@@ -79,9 +79,7 @@ public slots:
     void openSelectedTorrentsFolder() const;
     void recheckSelectedTorrents();
     void reannounceSelectedTorrents();
-    void setDlLimitSelectedTorrents();
-    void setUpLimitSelectedTorrents();
-    void setMaxRatioSelectedTorrents();
+    void setTorrentOptions();
```

Comment:
Try `void setTorrentOptions() const;`

Reply:
Can't, `parent` argument for TorrentOptionsDialog would have to be const and then i can't init `QDialog` with it...